src/control/controlindicatortimer.cpp

```diff
@@ -0,0 +1,32 @@
+#include "control/controlindicatortimer.h"
+
+#include "control/controlobject.h"
+#include "moc_controlindicatortimer.cpp"
+
+namespace mixxx {
+
+ControlIndicatorTimer::ControlIndicatorTimer(QObject* pParent)
+        : QObject(pParent),
+          m_pCOIndicator250millis(std::make_unique<ControlObject>(
+                  ConfigKey("[Master]", "indicator_250millis"))),
+          m_pCOIndicator500millis(std::make_unique<ControlObject>(
+                  ConfigKey("[Master]", "indicator_500millis"))),
+          m_toggleIndicator500millisOnNextTimeout(true) {
+    m_pCOIndicator250millis->setReadOnly();
+    m_pCOIndicator500millis->setReadOnly();
+    connect(&m_timer, &QTimer::timeout, this, &ControlIndicatorTimer::slotTimeout);
+    m_timer.start(250);
+}
+
+void ControlIndicatorTimer::slotTimeout() {
+    // The timeout uses an interval of 250ms, so the 250ms indicator is always updated.
+    m_pCOIndicator250millis->forceSet(1 - m_pCOIndicator250millis->get());
```

Comment:
This is an odd way to toggle a boolean value.
```suggestion
    m_pCOIndicator250millis->forceSet(!m_pCOIndicator250millis->toBool());
```

Reply:
...because storing a boolean as double is odd ;) But I agree, for the sake of readability use this less efficient variant.

Reply:
lol, we went full circle with this :D I think I prefer the original version then. That makes it clear that we're storing doubles.

```suggestion
    m_pCOIndicator250millis->forceSet(m_pCOIndicator250millis->toBool() ? 0.0 : 1.0);
```

Reply:
sure that works